README.md

```diff
@@ -2,46 +2,61 @@
 
 # Strauss – PHP Namespace Renamer
 
-A tool to prefix namespaces and classnames in PHP files to avoid autoloading collisions.
+A tool to prefix namespaces, classnames, and constants in PHP files to avoid autoloading collisions.
 
-A fork of [Mozart](https://github.com/coenjacobs/mozart/). For [Composer](https://getcomposer.org/) for PHP.
+A fork of [Mozart](https://github.com/coenjacobs/mozart/) for [Composer](https://getcomposer.org/) for PHP.
 
-The primary use case is WordPress plugins, where different plugins active in a single WordPress install could each include different versions of the same library. The version of the class loaded would be whichever plugin registered the autoloader first, and all subsequent instantiations of the class will use that version, with potentially unpredictable behaviour and missing functionality.    
+Have you ever activated a WordPress plugin that has a conflict with another because the plugins use two different versions of the same PHP library? **Strauss is the solution to that problem** - it ensures that _your_ plugin's PHP dependencies are isolated and loaded from your plugin rather than loading from whichever plugin's autoloader registers & runs first.
 
 > ⚠️ **Sponsorship**: I don't want your money. [Please write a unit test to help the project](https://brianhenryie.github.io/strauss/).
 
-## Breaking Changes
-
-* v0.16.0 – will no longer prefix PHP built-in classes seen in polyfill packages
-* v0.14.0 – `psr/*` packages no longer excluded by default
-* v0.12.0 – default output `target_directory` changes from `strauss` to `vendor-prefixed`
+## Table of Contents
 
-Please open issues to suggest possible breaking changes. I think we can probably move to 1.0.0 soon. 
+* [Installation](#installation)
+    * [As a `.phar` file](#as-a-phar-file-within-composerjson-recommended) (recommended)
+    * [As a dev dependency via composer](#as-a-dev-dependency-via-composer-not-recommended)  (not recommended)
+    * [Adding `composer dump-autoload`](#adding-composer-dump-autoload)
+* [Usage](#usage)
+* [Configuration](#configuration)
+* [Autoloading](#autoloading)
+* [Motivation & Comparison to Mozart](#motivation--comparison-to-mozart)
+* [Alternatives](#alternatives)
+* [Breaking Changes](#breaking-changes)
+* [Acknowledgements](#acknowledgements)
 
-## Use
+## Installation
 
-Require as normal with Composer:
+### As a `.phar` file (recommended)
```

Comment:
I think Phive, [phar.io](https://phar.io/), is the ideal way, but it's not how I use it myself so I'm hesitant to write documentation for something I don't do myself.

Also is using `bin` as the directory for downloaded tools convention? It is for the tools being distributed. Easily manageable with `.gitignore` as you're recommending.

https://github.com/BrianHenryIE/strauss/blob/3035f84066eac016db769494f08b327c6c41f60f/composer.json#L14